src/main/scala/uk/gov/nationalarchives/aggregate/processing/modules/AssetProcessing.scala

```diff
@@ -111,13 +111,22 @@ class AssetProcessing(s3Utils: S3Utils)(implicit logger: Logger) {
             )
             handleProcessError(error, s3Bucket, objectKey)
           } else {
+            val metadataConfig: ConfigUtils.MetadataConfiguration = ConfigUtils.loadConfiguration
```

Comment:
I would make these `private lazy val`s at the class level. Makes the method cleaner and easier to read.